gen3/bin/ecr.sh

```diff
@@ -179,14 +179,25 @@ gen3_ecr_update_policy() {
   aws ecr set-repository-policy --repository-name "$repoName" --policy-text "$policy"
 }
 
-
 #
 # List the `gen3/` repository names (in the current account)
 #
 gen3_ecr_repolist() {
   aws ecr describe-repositories | jq -r '.repositories[] | .repositoryName' | grep '^gen3/'
 }
 
+# Check if the Quay image exists in ECR repository
+gen3_describe_image() {
+    local repoName="$1"
+    shift
+    local imageName="$2"
```

Comment:
If this is the docker version tag we should make it clearer by calling it imageTag instead?

Reply:
I have renamed it to use `tagName` like the whole script uses